lib/debt_ceiling/debt.rb

```diff
@@ -4,36 +4,54 @@ class Debt
     extend Forwardable
     DoNotWhitelistAndBlacklistSimulateneously = Class.new(StandardError)
 
-    attr_reader :file_attributes
     def_delegators :file_attributes, :path, :analysed_module, :module_name, :linecount, :source_code
-    def_delegator :analysed_module, :rating
-    attr_accessor :debt_amount
-    def_delegator :debt_amount, :to_i
+    def_delegator  :analysed_module, :rating
```

Comment:
def_delegators is for assigning a bunch of methods to one object...  this is assigning rating to analysed_module, which is in turn delegated to file_attributes...  so either that's not an option, or I'm misunderstanding your question :-)